.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
You may have misunderstood my previous review feedback, I'm suggesting to add the entire .idea/ folder to the root /.gitignore, not to have a .gitignore inside the .idea/ folder. Does this make sense?

Reply:
Let me try this in a new branch